deploy/kustomize/patches/worker-subs-extraction.yaml

```diff
@@ -0,0 +1,196 @@
+          value: ${TRINO_HOST}
+        - name: TRINO_PORT
+          value: ${TRINO_PORT}
+        - name: PROMETHEUS_PUSHGATEWAY
+          value: ${PROMETHEUS_PUSHGATEWAY}
+        - name: SOURCES_API_PREFIX
+          value: ${SOURCES_API_PREFIX}
+        - name: UNLEASH_CACHE_DIR
+          value: ${UNLEASH_CACHE_DIR}
+        - name: WORKER_CACHE_TIMEOUT
+          value: ${WORKER_CACHE_TIMEOUT}
+        - name: QE_SCHEMA
+          value: ${QE_SCHEMA}
+        - name: ENHANCED_ORG_ADMIN
+          value: ${ENHANCED_ORG_ADMIN}
+      livenessProbe:
+        httpGet:
+          path: /livez
+          port: metrics
+          scheme: HTTP
+        initialDelaySeconds: 30
+        periodSeconds: 20
+        successThreshold: 1
+        failureThreshold: 5
+        timeoutSeconds: 10
+      readinessProbe:
+        httpGet:
+          path: /readyz
+          port: metrics
+          scheme: HTTP
+        initialDelaySeconds: 30
+        periodSeconds: 20
+        successThreshold: 1
+        failureThreshold: 5
+        timeoutSeconds: 10
+      resources:
+        limits:
+          cpu: ${WORKER_SUBS_EXTRACTION_CPU_LIMIT}
+          memory: ${WORKER_SUBS_EXTRACTION_MEMORY_LIMIT}
+        requests:
+          cpu: ${WORKER_SUBS_EXTRACTION_CPU_REQUEST}
+          memory: ${WORKER_SUBS_EXTRACTION_MEMORY_REQUEST}
+      volumeMounts:
+        - mountPath: /var/tmp/masu/
+          name: koku-worker-data
+        - name: gcp-credentials
+          mountPath: /etc/gcp
+          readOnly: true
+        - name: tmp-data
+          mountPath: ${TMP_DIR}
+      volumes:
+        - name: tmp-data
+          emptyDir: {}
+        - name: koku-worker-data
+          emptyDir: {}
+        - name: gcp-credentials
+          secret:
+            secretName: koku-gcp
+            items:
+              - key: gcp-credentials
+                path: gcp-credentials.json
+
+- op: add
+  path: /parameters/-
+  value:
+    displayName: Minimum replicas
+    name: WORKER_SUBS_EXTRACTION_REPLICAS
+    required: true
+    value: "0"
+- op: add
+  path: /parameters/-
+  value:
+    displayName: Memory Request
+    name: WORKER_SUBS_EXTRACTION_MEMORY_REQUEST
+    required: true
+    value: 300Mi
+- op: add
+  path: /parameters/-
+  value:
+    displayName: Memory Limit
+    name: WORKER_SUBS_EXTRACTION_MEMORY_LIMIT
+    required: true
+    value: 500Mi
+- op: add
+  path: /parameters/-
+  value:
+    displayName: CPU Request
+    name: WORKER_SUBS_EXTRACTION_CPU_REQUEST
+    required: true
+    value: 100m
+- op: add
+  path: /parameters/-
+  value:
+    displayName: CPU Limit
+    name: WORKER_SUBS_EXTRACTION_CPU_LIMIT
+    required: true
+    value: 200m
+- op: add
+  path: /parameters/-
+  value:
+    displayName: Worker Queue
+    name: WORKER_SUBS_EXTRACTION_WORKER_QUEUE
+    required: true
+    value: "subs_extraction"
+- op: add
+  path: /parameters/-
+  value:
+    displayName: SUBS Extraction debug
+    name: ENABLE_SUBS_DEBUG
+    value: "False"
```

Comment:
Rather than define this twice, we should define ENABLE_SUBS_DEBUG in the base.yaml and then use it in the worker patches.